addon/components/abstract-input.js

```diff
@@ -132,12 +132,11 @@ export default Component.extend(PropTypeMixin, {
 
     /**
      * Handle user updating value
-     * @param {Event} e - event
+     * @param {Event} data - event
      */
-    onChange (e) {
+    onChange (data) {
```

Comment:
It seems odd to name this argument `data` when the JSDoc description for it is `event`